vuepress/docs/next/tutorials/compose/page-management.md

```diff
@@ -1,82 +1,98 @@
 
 # Page Management
 
-The App Builder provides the capability to publish application pages containing content as well as other Entando components. Page Templates are used to layout the available frames on a page.
+The [Entando App Builder](../../docs/compose/app-builder.md) provides the capability to publish application pages containing content as well as other Entando components. Page Templates are used to define the page layout of available frames.
```

Comment:
Page Templates are used to define the page layout of available frames. >> don't templates define how the frames are laid out on a page, seems strange the other way around??

Reply:
yes... the page layout of the frames = the layout of the frames on a page, right?? i can switch back, it was just a bit wordy